internal/resource.go

```diff
@@ -92,6 +92,10 @@ func (l *Resource) Download(dir string, mode os.FileMode, ctx context.Context) e
 		// Download file in the target directory so that the call to
 		// os.Rename is atomic.
 		lpath, err := GetUrlToDir(u, dir, ctx)
+		defer func(path string) {
+			pathDir := filepath.Dir(path)
+			os.Remove(pathDir)
```

Comment:
Is there ever a case will this need to be `os.RemoveAll()`?

Reply:
That's a good idea. In case of error there could be leftovers in there. Changed.